ai/status.py

```diff
@@ -16,39 +16,16 @@ def read_version() -> str:
 
     return Path('.git/HEAD').read_text()
 
-
-class Status():
+async def report_status(context):
     '''
-    This module can give a short report on the AI.
+    Creates an embed with some debug-information about the AI.
```

Comment:
```suggestion
    Creates an embed with some debug information about the AI.
```
Yes I'm very nitpicky and want to grammar-correct comments as well. Despite all the HexCorp programming, I'm still a prick about this kind of stuff >_<